sys-cluster/teleport/files/teleport.confd

```diff
@@ -0,0 +1,30 @@
+# /etc/conf.d/teleport: config file for /etc/init.d/teleport
+
+# Where is your teleport.yaml file stored?
+
```

Comment:
Why are you using so much empty lines everywhere? It's horrible unreadable when even a simple file can't fit on a single screen.